test/arrays/userAPI/arrayOps2D-targetLocales.good

```diff
@@ -1,2 +1,105 @@
-./arrayAPItest.chpl:7: In function 'testArrayAPI2D':
-./arrayAPItest.chpl:92: error: targetLocales is unsupported by default domains
```

Comment:
Here, I was imagining that we'd actually take this out of the "error cases" category and make it part of the "normal" operation of the test, just to avoid maintenance costs and duplication in the .good files (though this'll mean updating most of the .goods in that directory).